jwst/spectral_leak/spectral_leak.py

```diff
@@ -41,21 +40,26 @@ def do_correction(sp_leak_ref, ch1b, ch3a):
 
     # Spectral leak vector
 
-    leak = np.interp(wave3a, 2*wave1b, spec1b) * np.interp(wave3a, leak_wave, leak_percent)
-    # Factor of 2 in 2*wavelb is  necessary because we're dealing with second order light.  Without this the interpolation simply
-    # uses the last point in the spectrum, which is close enough to correct as won't be noticeable for BAND spectra,
-    # but for CH spectra can be quite different.
+    leak = np.interp(wave3a, 2 * wave1b, spec1b) * np.interp(wave3a, leak_wave, leak_percent)
```

Comment:
Why move the comments after the code?  I think it's easier to read if the comments come before.